backend/src/main/kotlin/io/zell/zdb/log/ApplicationRecord.kt

```diff
@@ -15,8 +14,7 @@ class ApplicationRecord(val index: Long, val term : Long) : PersistedRecord {
     }
 
     override fun toString(): String {
-
-        val entriesJson = entries.map { it.toJson() }.joinToString()
+        val entriesJson = entries.map { "\"${it.toString().replace("\"", "'")}\"" }.joinToString()
```

Comment:
:question: Why this change? :thinking: toJson should still work?

Reply:
Oops, that's my bad. It was left over when I tried to get it running on 8.2 logs without actually doing the upgrade. Good catch :+1:

Reply:
I reverted this change